### PR TITLE
834573: Library Bounds to Viewer Bounds sample

### PR DESCRIPTION
**Pull request by SF4524LogeshKumar**
Description:
Created Sample on "How to convert Library Bounds to Viewer Bounds" in EJ2 PDF Viewer

### CODE REVIEW

File: How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/Pages/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+@model IndexModel
+@{
+    ViewData["Title"] = "Home page";
+}
+
+<div class="text-center">
+    <ejs-pdfviewer id="pdfviewer"
+                   style="height:600px"
+                   documentPath="https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf"
+                   serviceUrl="https://services.syncfusion.com/js/production/api/pdfviewer">
```
Comment (rparthi-pdf) [inline How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/Pages/Index.cshtml:11]: @SF4524LogeshKumar  - Why we have used production service URL in ASP.NET core sample. Please check it and also PDF Viewer server library package was missed in the project reference.

**Review by rparthi-pdf — CHANGES_REQUESTED** [review]

File: How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/Pages/Index.cshtml (hunk 1 above)
Comment (SF4524LogeshKumar) [reply How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/Pages/Index.cshtml]: Addressed changes

File: How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/PDFViewerSample.csproj [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.EJ2.AspNet.Core" Version="*" />
+    <PackageReference Include="Syncfusion.EJ2.PdfViewer.AspNet.Core.Windows" Version="*" />
```
Comment (rparthi-pdf) [inline How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/PDFViewerSample.csproj:11]: @SF4524LogeshKumar Please use the common nuget package in the sample.

Comment (SF4524LogeshKumar) [reply How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/PDFViewerSample.csproj]: updated common packages

**Review by rparthi-pdf — APPROVED** [review]

**Pull request merged**